p/protobuf/build_info.json

```diff
@@ -0,0 +1,29 @@
+{
+  "package_name" : "protobuf",
+  "github_url": "https://github.com/protocolbuffers/protobuf",
+  "version": "v21.5",
+  "default_branch": "main",
+  "package_dir": "p/protobuf/",
+  "docker_cmd" :"docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+  "build_script": "protobuf_ubi_8.5_v21.5.sh",
+
+  "v21.5" : {
+  "build_script":"protobuf_ubi_8.5_v21.5.sh"
```

Comment:
@muskaan2199 if we are using same build script across version, can we combine into one entry. 
 you can just mention different one only in regex block and common one can go in default section